# ###########################################################################
# #
# # Copyright 2026 Samsung Electronics All Rights Reserved.
# #
# # Licensed under the Apache License, Version 2.0 (the "License");
# # you may not use this file except in compliance with the License.
# # You may obtain a copy of the License at
# #
# # http://www.apache.org/licenses/LICENSE-2.0
# #
# # Unless required by applicable law or agreed to in writing,
# # software distributed under the License is distributed on an
# # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
# # express or implied. See the License for the specific language
# # governing permissions and limitations under the License.
# #
# # ###########################################################################

From 77955874b3c97f1db2bc261ce3ddb76873cdaab7 Mon Sep 17 00:00:00 2001
From: guohao15 <guohao15@xiaomi.com>
Date: Mon, 8 May 2023 20:19:37 +0800
Subject: [PATCH 15/21] test:ltp fix ltp_interfaces_sigaction_23_10 deadloop at
 second run

VELAPLATFO-7303

Signed-off-by: guohao15 <guohao15@xiaomi.com>
Change-Id: Ib441191da3e3380ed5e54b2914ffec8d3307b173
---
 .../conformance/interfaces/sigaction/23-10.c                    | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/testcases/open_posix_testsuite/conformance/interfaces/sigaction/23-10.c b/testcases/open_posix_testsuite/conformance/interfaces/sigaction/23-10.c
index 9274e1154..8a0039f14 100644
--- a/testcases/open_posix_testsuite/conformance/interfaces/sigaction/23-10.c
+++ b/testcases/open_posix_testsuite/conformance/interfaces/sigaction/23-10.c
@@ -123,6 +123,8 @@ int main(void)
 
 	struct sigaction sa;
 
+        called = 0;
+
 	/* Initialize output */
 	output_init();
 
-- 
2.40.1

